src/components/action-popover/action-popover.spec.js

```diff
@@ -209,6 +209,14 @@ describe('ActionPopover', () => {
     }).not.toThrow();
   });
 
+  it('renders with the menu aligned to the right if the "rightAlignMenu" prop is true', () => {
+    render({ rightAlignMenu: true });
+    const { menu } = getElements();
+    assertStyleMatch({
+      right: undefined
```

Comment:
I think this should be `right: 0`

Reply:
The assertion is correct but my description is vague, I've updated the test and add more to the assertion to hopefully make it clearer 😄